tools/prepare_prebuilt_workers/prepare_prebuilt_workers.go

```diff
@@ -107,7 +107,7 @@ func main() {
 		split := strings.Split(pair, ":")
 
 		if len(split) != 2 || split[len(split)-1] == "" {
-			log.Fatalf("Input error in language and gitref selection, please follow the format as language:gitref, for example: cxx:<commit-sha>")
+			log.Fatalf("Input error in language and gitref selection, please follow the format as language:gitref, for example: c++:<commit-sha>")
```

Comment:
Are we trying update cxx to c++? If so I will create another PR to update everything from cxx to c++. I think if we only update it here, it might cause some script to not work.

Reply:
We cannot update cxx to c++ because the character "+" cannot be used in labels. This is the only language we will always need to translate. But we accept c++ in the value of flags, and we should, because that is the language name used in scenario generation.

Reply:
Right! I forgot you test cxx before, so in this case everything in test-infra stays as `cxx`.